app/views/staff/program_sessions/_session_row_active.html.haml

```diff
@@ -1,11 +1,8 @@
 %tr{ data: { 'session-id' => program_session.id } }
   %td= link_to(program_session.title,
       event_staff_program_session_path(program_session.event, program_session))
-  %td= program_session.speaker_names
-  %td= program_session.speaker_emails
-  %td labels~
-  -#= program_session.review_tags_labels
+  %td= program_session.speakers.map { |speaker| link_to speaker.name, event_staff_program_speaker_path(program_session.event, speaker) }.join(", ").html_safe
```

Comment:
Yep, think of decorators as model-based helpers so yes any union of display/view code with model code is appropriate, including path helpers or other global helper methods.